redaxo/src/core/tests/console/config/get_test.php

```diff
@@ -37,4 +37,15 @@ public function testKeyNotFound()
         ]);
         static::assertEquals(1, $commandTester->getStatusCode());
     }
+
+    public function testPackageKeyFound()
```

Comment:
diesen test den ich im urspr. PR gefixed hatte, gibts wohl im bugfix branch nicht. daher ist er hier als "added" markiert

Reply:
urgs, das hatte ich nicht bedacht. die option, die hier getestet wird, gibt es ja noch gar nicht in bugfix, oder?

Reply:
ja, sehs grad

Reply:
dann lass es uns der einfachheit halber doch in master mergen, oder?

Reply:
hab jetzt in 2 PRs aufgeteilt

Reply:
jupp ok